### ISSUE TITLE
Input "guidance" is not accessible to screen readers.

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1183]
The guidance popover generated via the use of the input `guidance` option is not accessible to screen readers. The trigger cannot be accessed via the keyboard and the guidance text is not announced on input focus.

We could look at replacing with the new accessible tooltips, with a slightly different styling.

![image](https://user-images.githubusercontent.com/756393/77065762-d294ac80-69d9-11ea-8730-f66910265791.png)

**Comment by Stanton** [comment]
We should consider retiring this feature as I don't believe it is particularly well used.

**Comment by Stanton** [comment]
# Instances in production

**CMS**
* Marketing registration questions

**XFP**
* Form publish UI
* Received submissions

**Comment by Stanton** [comment]
@jamesjacobs Can you find the instances in CXM? I searched either for `guidance'` `guidance"`, `guidance:` and `input-group-guidance`.

**Comment by jamesjacobs** [comment]
**CXM**

- Main search (search tips)
- Case type create
- Case type edit
- Integration server settings
